packages/openfeature-browser-provider/example/components/HuddleFeature.tsx

```diff
@@ -1,19 +1,37 @@
 "use client";
 
 import React from "react";
-import { useBooleanFlagValue } from "@openfeature/react-sdk";
+import {
+  useBooleanFlagValue,
+  useObjectFlagDetails,
+} from "@openfeature/react-sdk";
 import { track } from "@/app/featureManagement";
 
 const featureKey = "huddle";
 
 export const HuddleFeature = () => {
   const isEnabled = useBooleanFlagValue(featureKey, false);
+  const { variant: provider, value: config } = useObjectFlagDetails(
```

Comment:
`provider` is a little confusing because the terminology is overloaded in OpenFeature land. Maybe "huddleServiceProvider" or something?